### PR TITLE
feat(api): Add zeppelinos well-known slots

### PR DESCRIPTION
**Pull request by perekopskiy**
## What ❔

Add zeppelinos well-known slots

## Why ❔

Makes possible to use trusted zeppelinos proxy during tx validation

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.
- [ ] Spellcheck has been run via `zk spellcheck`.

### CODE REVIEW

File: core/lib/constants/src/trusted_slots.rs [hunk 1]
```diff
@@ -31,12 +31,24 @@ const INITIALIZER_INITIALING_SLOT: H256 = H256([
     0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
 ]);
 
+const ZEPPELINOS_ADMIN_SLOT: H256 = H256([
```
Comment (vladbochok) [inline core/lib/constants/src/trusted_slots.rs:34]: ZEPPELINOS?

Comment (perekopskiy) [reply core/lib/constants/src/trusted_slots.rs:34]: Yes, I didn't find an ERC for that, but it's used here:
https://github.com/zeppelinos/zos-lib/blob/8a16ef3ad17ec7430e3a9d2b5e3f39b8204f8c8d/contracts/upgradeability/UpgradeabilityProxy.sol#L24
https://github.com/zeppelinos/zos-lib/blob/8a16ef3ad17ec7430e3a9d2b5e3f39b8204f8c8d/contracts/upgradeability/AdminUpgradeabilityProxy.sol#L26
and circle's FiatTokenProxy is almost the same. If you think another name is more suitable please suggest

Comment (vladbochok) [reply core/lib/constants/src/trusted_slots.rs:34]: ZEPPELINOS is the legacy name for OPEN_ZEPPELIN library, I think we can change the name to OPEN_ZEPPELIN_ADMIN_SLOT

Comment (vladbochok) [reply core/lib/constants/src/trusted_slots.rs:34]: Uh, I see, the slot is also legacy one

**Review by vladbochok — APPROVED** [review]

**Pull request merged**